test: feature_dbcrash listunspent times out

On current master


```
2019-07-29T16:43:20.074000Z TestFramework (INFO): Iteration 31, generating 2500 transactions [15, 3, 0]
2019-07-29T16:45:53.858000Z TestFramework (ERROR): JSONRPC error
Traceback (most recent call last):
  File "/root/workspace/bitcoin-core/test/functional/test_framework/authproxy.py", line 163, in _get_response
    http_response = self.__conn.getresponse()
  File "/usr/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/usr/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.6/http/client.py", line 258, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/lib/python3.6/socket.py", line 586, in readinto
    return self._sock.recv_into(b)
socket.timeout: timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/workspace/bitcoin-core/test/functional/test_framework/test_framework.py", line 193, in main
    self.run_test()
  File "./test/functional/feature_dbcrash.py", line 250, in run_test
    utxo_list = self.nodes[3].listunspent()
  File "/root/workspace/bitcoin-core/test/functional/test_framework/coverage.py", line 47, in __call__
    return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)
  File "/root/workspace/bitcoin-core/test/functional/test_framework/authproxy.py", line 139, in __call__
    response, status = self._request('POST', self.__url.path, postdata.encode('utf-8'))
  File "/root/workspace/bitcoin-core/test/functional/test_framework/authproxy.py", line 107, in _request
    return self._get_response()
  File "/root/workspace/bitcoin-core/test/functional/test_framework/authproxy.py", line 170, in _get_response
    self.__conn.timeout)})
test_framework.authproxy.JSONRPCException: 'listunspent' RPC took longer than 90.000000 seconds. Consider using larger timeout for calls that take longer to return. (-344)
2019-07-29T16:45:54.033000Z TestFramework (INFO): Stopping nodes
2019-07-29T16:45:54.060000Z TestFramework.node3 (ERROR): Unable to stop node.
Traceback (most recent call last):
  File "/root/workspace/bitcoin-core/test/functional/test_framework/test_node.py", line 264, in stop_node
    self.stop(wait=wait)
  File "/root/workspace/bitcoin-core/test/functional/test_framework/coverage.py", line 47, in __call__
    return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)
  File "/root/workspace/bitcoin-core/test/functional/test_framework/authproxy.py", line 139, in __call__
    response, status = self._request('POST', self.__url.path, postdata.encode('utf-8'))
  File "/root/workspace/bitcoin-core/test/functional/test_framework/authproxy.py", line 106, in _request
    self.__conn.request(method, path, postdata, headers)
  File "/usr/lib/python3.6/http/client.py", line 1239, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.6/http/client.py", line 1250, in _send_request
    self.putrequest(method, url, **skips)
  File "/usr/lib/python3.6/http/client.py", line 1108, in putrequest
    raise CannotSendRequest(self.__state)
http.client.CannotSendRequest: Request-sent
2019-07-29T16:46:55.268000Z TestFramework.utils (ERROR): wait_until() failed. Predicate: ''''
    def is_node_stopped(self):
        """Checks whether the node has stopped.

        Returns True if the node has stopped. False otherwise.
        This method is responsible for freeing resources (self.process)."""
        if not self.running:
            return True
        return_code = self.process.poll()
        if return_code is None:
            return False

        # process has stopped. Assert that it didn't return an error code.
        assert return_code == 0, self._node_msg(
            "Node returned non-zero exit code (%d) when stopping" % return_code)
        self.running = False
        self.process = None
        self.rpc_connected = False
        self.rpc = None
        self.log.debug("Node stopped")
        return True
'''
Traceback (most recent call last):
  File "./test/functional/feature_dbcrash.py", line 273, in <module>
    ChainstateWriteCrashTest().main()
  File "/root/workspace/bitcoin-core/test/functional/test_framework/test_framework.py", line 218, in main
    self.stop_nodes()
  File "/root/workspace/bitcoin-core/test/functional/test_framework/test_framework.py", line 401, in stop_nodes
    node.wait_until_stopped()
  File "/root/workspace/bitcoin-core/test/functional/test_framework/test_node.py", line 305, in wait_until_stopped
    wait_until(self.is_node_stopped, timeout=timeout)
  File "/root/workspace/bitcoin-core/test/functional/test_framework/util.py", line 225, in wait_until
    raise AssertionError("Predicate {} not true after {} seconds".format(predicate_source, timeout))
AssertionError: Predicate ''''
    def is_node_stopped(self):
        """Checks whether the node has stopped.

        Returns True if the node has stopped. False otherwise.
        This method is responsible for freeing resources (self.process)."""
        if not self.running:
            return True
        return_code = self.process.poll()
        if return_code is None:
            return False

        # process has stopped. Assert that it didn't return an error code.
        assert return_code == 0, self._node_msg(
            "Node returned non-zero exit code (%d) when stopping" % return_code)
        self.running = False
        self.process = None
        self.rpc_connected = False
        self.rpc = None
        self.log.debug("Node stopped")
        return True
''' not true after 60 seconds
[node 3] Cleaning up leftover process

--- a/test/functional/feature_dbcrash.py
+++ b/test/functional/feature_dbcrash.py
@@ -30,17 +30,27 @@ import http.client
 import random
 import time
 
-from test_framework.messages import COIN, COutPoint, CTransaction, CTxIn, CTxOut, ToHex
+from test_framework.messages import (
+    COIN,
+    COutPoint,
+    CTransaction,
+    CTxIn,
+    CTxOut,
+    ToHex,
+)
 from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import assert_equal, create_confirmed_utxos, hex_str_to_bytes
+from test_framework.util import (
+    assert_equal,
+    create_confirmed_utxos,
+    hex_str_to_bytes,
+)
 
 
 class ChainstateWriteCrashTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 4
         self.setup_clean_chain = False
-        # Need a bit of extra time for the nodes to start up for this test
-        self.rpc_timeout = 90
+        self.rpc_timeout = 180
 
         # Set -maxmempool=0 to turn off mempool memory sharing with dbcache
         # Set -rpcservertimeout=900 to reduce socket disconnects in this

--- a/test/functional/feature_dbcrash.py
+++ b/test/functional/feature_dbcrash.py
@@ -64,7 +64,8 @@ class ChainstateWriteCrashTest(BitcoinTestFramework):
         self.node2_args = ["-dbcrashratio=24", "-dbcache=16"] + self.base_args
 
         # Node3 is a normal node with default args, except will mine full blocks
-        self.node3_args = ["-blockmaxweight=4000000"]
+        # and non-standard txs (e.g. txs with "dust" outputs)
+        self.node3_args = ["-blockmaxweight=4000000", "-acceptnonstdtxn"]
         self.extra_args = [self.node0_args, self.node1_args, self.node2_args, self.node3_args]
 
     def skip_test_if_missing_module(self):

--- a/test/functional/feature_fee_estimation.py
+++ b/test/functional/feature_fee_estimation.py
@@ -120,9 +120,16 @@ def check_estimates(node, fees_seen):
         else:
             assert_greater_than_or_equal(i + 1, e["blocks"])
 
+
 class EstimateFeeTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 3
+        # mine non-standard txs (e.g. txs with "dust" outputs)
+        self.extra_args = [
+            ["-acceptnonstdtxn", "-maxorphantx=1000", "-whitelist=127.0.0.1"],
+            ["-acceptnonstdtxn", "-blockmaxweight=68000", "-maxorphantx=1000"],
+            ["-acceptnonstdtxn", "-blockmaxweight=32000", "-maxorphantx=1000"],
+        ]
 
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
@@ -133,9 +140,7 @@ class EstimateFeeTest(BitcoinTestFramework):
         But first we need to use one node to create a lot of outputs
         which we will use to generate our transactions.
         """
-        self.add_nodes(3, extra_args=[["-maxorphantx=1000", "-whitelist=127.0.0.1"],
-                                      ["-blockmaxweight=68000", "-maxorphantx=1000"],
-                                      ["-blockmaxweight=32000", "-maxorphantx=1000"]])
+        self.add_nodes(3, extra_args=self.extra_args)
         # Use node0 to mine blocks for input splitting
         # Node1 mines small blocks but that are bigger than the expected transaction rate.
         # NOTE: the CreateNewBlock code starts counting block weight at 4,000 weight,

